### DISCUSSION TITLE
[informative / not an issue] NeRF-compatible dataset paper published.

### DISCUSSION THREAD

**Discussion by mcourteaux** [NVlabs/instant-ngp#547] [Show and tell]
Hi @Tom94 and others. We just published a paper presenting a publicly-available dataset ("SILVR") for large volumes of fully immersive (i.e. 360°) content, which is compatible with NeRF. We also demonstrate the use of the dataset with instant-ngp. Our dataset uses 180° fisheye projections, which we transform to rectilinear and then feed into instant-ngp. You might be interested in checking out the landing page for a quick look at the work.

https://idlabmedia.github.io/large-lightfields-dataset/ (or the instant-ngp specific section: https://idlabmedia.github.io/large-lightfields-dataset/#nerf-configuration-generator)

If you like it, feel free to link it in your README as a possible source for a scene.

Thanks for your great high-quality software which demonstrates the raw power of hardware these days. I greatly respect your work.

**Comment by Tom94** [comment]
Hey Martijn, very nice work with SILVR! I didn't yet have the time to take a closer look and play around with the scenes, but they look really interesting. I loved the barbershop result you've previously shared in another thread.

When I had a chance to play around with the datasets, I'll get back to you about linking in the readme. Congrats again!

Cheers!

**Comment by mcourteaux** [comment]
By the way, as this dataset is fully made of fisheye projections, it would be cool if instant-ngp can use these images directly to convert them into rays, instead of assuming rectilinear projection.

PS: paper is available here: https://arxiv.org/abs/2204.09523

**Reply by KamiCalcium** [reply]
Any follow up on this? Fisheye images directly convert to ray sounds super interesting and helpful!

**Reply by Tom94** [reply]
Sure, this is just a bit of plumbing. I had tried directly fitting fisheye lens models in the past (using COLMAP's OpenCV fisheye lens support) and had mixed results. Going to push the corresponding code throughout the day. PR is here https://github.com/NVlabs/instant-ngp/pull/1147

**Comment by mcourteaux** [comment]
Ooh, I just noticed you added the dataset to the README. Super cool. Thanks for taking the time to have a look at it :)
